grails-app/services/org/pih/warehouse/forecasting/ForecastingService.groovy

```diff
@@ -43,12 +44,11 @@ class ForecastingService {
             def monthlyDemand = totalDemand / Math.floor((demandPeriod / 30))
```

Comment:
Change the calculation here and use the variable in the returned map

Reply:
@awalkowiak as I wrote, firstly I've done that, but the "On hand months" wasn't working correctly then (I attached two screenshots about that - one when using variable in the returned map and second when using calculation inside returned map)

Reply:
Then you should use `monthlyDemand` in the `new BigDecimal(monthlyDemand).setScale(0, RoundingMode.HALF_UP)`

Reply:
yeah, that's a big inadvertence from my side and unnecessary redundancy from my side, will change it